app/src/main/kotlin/com/simplemobiletools/calculator/activities/MoneyActivity.kt

```diff
@@ -126,4 +131,22 @@ class MoneyActivity : SimpleActivity(), Calculator , TaxCalculator {
 
     private fun getMoneyButtonIds() = arrayOf(btn_tip, btn_currency, btn_taxes)
     private fun getButtonIds() = arrayOf(btn_decimal, btn_0, btn_1, btn_2, btn_3, btn_4, btn_5, btn_6, btn_7, btn_8, btn_9)
-}
+
+
+    private fun displayTipsDialog() {
+        val numbers = arrayOf("10%", "12%", "15%", "18%", "20%", "25%")
+
+        val tipDialog = AlertDialog.Builder(this@MoneyActivity)
+        tipDialog.setTitle("Calculate Tip")
+
+        tipDialog.setItems(numbers, DialogInterface.OnClickListener { _, index ->
+            var tipPercentage = numbers[index].trimEnd('%').toDouble() / 100
```

Comment:
Why don't you offload this computation of to the tipOperation class ?

Reply:
Because from logical point of view I feel like the tipOperation class takes a percentage - read: decimal/Double - to perform its operations. Having the Operation convert a string such as '15%' to 0.15 doesn't seem to make sense.

Reply:
Yeah it could be reusable. I see both or you opinions as valid. Let's keep it as is

Reply:
Fair Enough